src/main/java/com/blackducksoftware/integration/hub/dataservice/license/ComplexLicenseParser.java

```diff
@@ -55,25 +55,6 @@ public String parse(ComplexLicenseItem complexLicense) {
                 if (i < complexLicense.getWrappedComplexLicenseItems().size()) licenseText.append(operator);
                 i++;
             }
-
```

Comment:
I don't really object to this implementation, though I don't find it immediately obvious what it is doing. Perhaps we'll keep the api the same and revise it later. I do, however, object to a toString() invocation messing with object state. I'd suggest toString() be an idempotent operation.